cfgov/tccp/jinja2/tccp/macros/filter_form.html

```diff
@@ -58,7 +58,7 @@
 
 {%- macro filter_form(form) -%}
 
-<form action="." method="get">
+<form action="." method="get" hx-boost="true" hx-trigger="change delay:1s">
```

Comment:
I added a 1s delay so that the user doesn't get interrupted if they're trying to check multiple checkboxes in rapid succession.